src/structstore_shared.hpp

```diff
@@ -44,116 +50,138 @@ class StructStoreShared {
         ~SharedData() = delete;
     };
 
-    std::string shm_path;
-    int shm_fd;
-    SharedData* shm_ptr;
-    bool owning;
+    std::string path;
+    int fd;
+    SharedData* ptr;
+    bool reinit;
```

Comment:
It seems this is not needed as a member, it is only used in the constructor

Reply:
True, removed.